peer/mruinvmap.go

```diff
@@ -15,13 +16,17 @@ import (
 // MruInventoryMap provides a map that is limited to a maximum number of items
```

Comment:
Please call out this is a concurrent safe map (since the changes have made that true).